databroker/src/grpc/kuksa_val_v2/val.rs

```diff
@@ -793,17 +794,8 @@ async fn publish_values(
 
     // TODO check if provider is allowed to update the entries for the provided signals?
```

Comment:
Noticed the TODO - Is it so that we currently do not check token, even if broker is configured to require token?
If so, something we better should fix before release - not necessarily part of this PR?

Reply:
But we have `let broker = broker.authorized_access(&permissions);` earlier, so it is "broker" that does the check on the line below. Could you confirm @argerus